usaspending_api/awards/tests/test_awards.py

```diff
@@ -134,3 +134,19 @@ def test_award_date_signed_fy(client):
     for result in resp.data['results']:
         assert 'date_signed__fy' in result
         assert int(result['date_signed__fy']) > 2012
+
+
+@pytest.mark.django_db
+def test_manual_hash_eq_fain():
+    """test that records with equal FAIN hash as equal"""
+    m1 = mommy.make('awards.award', fain='ABC')
```

Comment:
Thanks for accompanying these changes with tests! Do you think that using `_fill_optional` parameter on `mommy.make` would be a simple way to get a more robust test (_i.e._, it would test that random noise in the non-hashed fields would not affect the way we need the hash to work). Feel free to tell me I'm over-thinking this!